pvlib/tests/test_modelchain.py

```diff
@@ -411,6 +412,26 @@ def test_run_model_from_irradiance_arrays_no_loss(
         mc_both.results.dc[1],
         mc_two.results.dc
     )
+    # repeat test with tuple/list of weather
+    mc_both_results = mc_both.results.copy()
```

Comment:
`ModelChainResults` doesn't have a `copy()` method. We will need to copy each of the elements of `mc_both.results.dc` individually.

Reply:
wishful thinking on my part